From 89f74fdc2893a68d2baba8d13cff4846d92840e8 Mon Sep 17 00:00:00 2001
From: Chris Kuehl <ckuehl@ocf.berkeley.edu>
Date: Sat, 20 Sep 2014 21:00:11 +0000
Subject: [PATCH] Fix ldlinux.c32 symlink path

The existing symlink creates a target path like:
debian-installer/amd64/debian-installer/amd64/boot-screens//ldlinux.c32

(note the duplicate of "d-i/amd64/" and the double slash after boot-screens)
---
 build/config/x86.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/x86.cfg b/build/config/x86.cfg
index 07f99b8..0bcd2a9 100644
--- a/build/config/x86.cfg
+++ b/build/config/x86.cfg
@@ -393,7 +393,7 @@ arch_netboot_dir: x86_syslinux
 	ln -sf $(NETBOOT_PATH)/pxelinux.0 $(TEMP_NETBOOT_DIR)/pxelinux.0
 	# This link is required because pxelinux only looks in the tftp root
 	# for this library (it does a PATH search for any others).
-	ln -sf $(NETBOOT_PATH)/$(BOOT_SCREEN_DIR)/ldlinux.c32 $(TEMP_NETBOOT_DIR)/ldlinux.c32
+	ln -sf $(BOOT_SCREEN_DIR)ldlinux.c32 $(TEMP_NETBOOT_DIR)/ldlinux.c32
 
 	set -e; \
 	$(foreach file,$(wildcard $(TEMP_BOOT_SCREENS)/*.txt), \
-- 
2.1.0

